modules/effects/src/actions.ts

```diff
@@ -25,8 +25,9 @@ export class Actions<V = Action> extends Observable<V> {
  * 'ofType' filters an Observable of Actions into an observable of the actions
  * whose type strings are passed to it.
  *
- * For example, `actions.pipe(ofType('add'))` returns an
- * `Observable<AddtionAction>`
+ * For example, if `actions` has type `Actions<Addition|Substraction>`, and
```

Comment:
Shouldn't this be `AdditionAction|SubtractionAction` or `Observable<Addition>`?

Reply:
Should it? I'm barely tipping my feet in, but my intention was to expand on [this](https://github.com/ngrx/platform/pull/1550/files#diff-37851f2da2623baafce63a5976e8d86bR36),
> as long as your `actions` object starts with a `Actions<SomeUnionOfActions>` instead of generic `Actions`.

Reply:
Gotcha. It should be `Actions<AdditionAction|SubstractionAction>` to be consistent with the provided text.